api/src/main/java/org/openmrs/api/context/Context.java

```diff
@@ -185,7 +185,7 @@ public static void setDAO(ContextDAO dao) {
 	/**
 	 * Spring init method that sets the authentication scheme.
 	 */
-	static private void setAuthenticationScheme() {
+	private static  void setAuthenticationScheme() {
```

Comment:
You've fixed the spaces at the beginning of the line nicely, but I suggest you turn on whitespace visibility in your ide.

![image](https://user-images.githubusercontent.com/9057882/92354091-efd14a80-f0e1-11ea-95f3-6921d747f5ea.png)
Do you see the two dots between the words "static" and "void"? One dot is one space, and there are two. ;)